changelog.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.1] - 2018-03-21
```

Comment:
Should probably give this a minor version bump as this would cause projects to fail if they don't add the new method?

Reply:
Good shout @m2de, I've now updated accordingly.